tests/bluetooth/controller/ctrl_hci/src/main.c

```diff
@@ -131,13 +137,12 @@ void test_hci_feature_exchange_wrong_handle(void)
 
 	zassert_equal(err, BT_HCI_ERR_UNKNOWN_CONN_ID, "Wrong reply for wrong handle\n");
 
+	/* Use up all local procedure contexts */
 	ctx_counter = 0;
 	do {
 		ctx = llcp_create_local_procedure(PROC_FEATURE_EXCHANGE);
 		ctx_counter++;
 	} while (ctx != NULL);
-	zassert_equal(ctx_counter, test_ctx_buffers_cnt() + 1,
```

Comment:
remove?

Reply:
Removal of this check is ok, as the point of the test is to confirm that after 'using all resources' an attempt to initiate the procedure will fail with CMD_DISALLOWED. And after splitting resource into local/remote there is no published API for getting the number of local procedure ctx allocated, so this check is simply removed